### DISCUSSION TITLE
Can hbm 1.12.2 extended be compatible with open computers mod?

### DISCUSSION THREAD

**Discussion by some525242** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1284] [Q&A]
Is it possible to integrate open computers mod with hbm mod 1.12.2 extended?

**Comment by F1xGOD** [comment]
damn, nobody answered....

**Comment by hz0909adc** [comment]
I think you should  go to 1.12.2HBM-ex github ask this question:-(
